dali/operators/reader/parser/caffe2_parser.h

```diff
@@ -189,7 +189,10 @@ class Caffe2Parser : public Parser<Tensor<CPUBackend>> {
     caffe2::TensorProtos protos;
     int consumed_inputs = 0;
 
-    DALI_ENFORCE(protos.ParseFromArray(data.data<uint8_t>(), data.size()));
+    DALI_ENFORCE(protos.ParseFromArray(data.data<uint8_t>(), data.size()),
+      make_string("Error parsing Caffe2 file: ", data.GetSourceInfo(),
+                  " (raw data length ", data.size(), ")"));
```

Comment:
And maybe, but it's up to you:
```suggestion
                  " (raw data length: ", data.size(), " bytes)."));
```